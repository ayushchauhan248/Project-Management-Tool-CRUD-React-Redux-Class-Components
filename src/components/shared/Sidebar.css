#sidebar {
  width: 200px;
  height: 100%;
  background-color: rgb(80, 73, 119);
  position: fixed;
  z-index: 1;
  top: 50px;
  left: 0;
  overflow-x: hidden;
  border-top: 1px solid white;
}
#lli{
  padding: 10px;
  color : white;
  font-family: 'Times New Roman', Times, serif;
  font-weight: 700;
}

.doFlex{
  display: flex;
  flex-direction: row;
}

#llii{
  padding: 12px;
  color: white;
}

.buttonLogout{
  background-color: rgb(80, 73, 119);
  width: 100%;
  border-right: 1px solid white;
}