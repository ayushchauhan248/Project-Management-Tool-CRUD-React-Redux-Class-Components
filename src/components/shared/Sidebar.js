import "./Sidebar.css";
import { Link } from "react-router-dom";
// import { MdCreateNewFolder } from "react-icons/md";
import { ImHome } from "react-icons/im";

import React, { Component } from "react";

class Sidebar extends Component {
  render() {
    return (
      <div id="sidebar">
        <div id="uul">
          <Link to="/dashboard" id="llnk" className="doFlex">
            <ImHome id="llii"></ImHome>
            <div id="lli">DASHBOARD</div>
          </Link>
        </div>
      </div>
    );
  }
}
export default Sidebar;
