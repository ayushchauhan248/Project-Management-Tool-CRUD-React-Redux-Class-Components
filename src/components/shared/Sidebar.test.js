import React from "react";
import Sidebar from "./Sidebar";
import { shallow, configure } from "enzyme";
import Adapter from "@wojtekmaj/enzyme-adapter-react-17";
configure({ adapter: new Adapter() });

it("should render snapshot of navbar", () => {
  const component = shallow(<Sidebar />);
  expect(component).toMatchSnapshot();
});
