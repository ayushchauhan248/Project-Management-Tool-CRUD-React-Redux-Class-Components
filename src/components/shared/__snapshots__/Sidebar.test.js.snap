// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render snapshot of navbar 1`] = `
<div
  id="sidebar"
>
  <div
    id="uul"
  >
    <Link
      className="doFlex"
      id="llnk"
      to="/dashboard"
    >
      <ImHome
        id="llii"
      />
      <div
        id="lli"
      >
        DASHBOARD
      </div>
    </Link>
  </div>
</div>
`;
